v0.11.0 Release Tour.ipynb

```diff
@@ -0,0 +1,229 @@
+    "        \n",
+    "        # You can also change multiple arguments at once:\n",
+    "        Circle.set_default(color=WHITE, fill_opacity=0.5)\n",
+    "        c = Circle().next_to(t, DOWN)\n",
+    "        VGroup(t, c).center()\n",
+    "        \n",
+    "        self.add(t, c)\n",
+    "\n",
+    "        # Call the method without arguments to restore the default behavior!\n",
+    "        Text.set_default()\n",
+    "        Circle.set_default()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dependent-cookie",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### [#2094](https://github.com/ManimCommunity/manim/pull/2094): Implicit function plotting\n",
+    "\n",
+    "There is now a new Mobject allowing to plot points that satisfy some equation. Here is an example for the curves defined by $xy^2 - x^2y - 2 = 0$ (yellow) and $x^3 - x + 1 - y^2 = 0$ (red)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "portuguese-barrier",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING ImplicitFunctionDemo\n",
+    "\n",
+    "class ImplicitFunctionDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        plane = NumberPlane()\n",
+    "        curve = ImplicitFunction(\n",
+    "            lambda x, y: x * y**2 - x**2 * y - 2,\n",
+    "            color=YELLOW,\n",
+    "        )\n",
+    "        self.add(plane)\n",
+    "        self.play(Create(curve))\n",
+    "        self.wait()\n",
+    "        elliptic_curve = ImplicitFunction(\n",
+    "            lambda x, y: x**3 - x + 1 - y**2,\n",
+    "            color=RED,\n",
+    "        )\n",
+    "        self.play(Create(elliptic_curve))\n",
+    "        self.wait()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "hourly-aerospace",
+   "metadata": {},
+   "source": [
+    "\n",
+    "### [#1991](https://github.com/ManimCommunity/manim/pull/1991): Added support for boolean operations on `VMobject`s\n",
+    "\n",
+    "Manim can now compute the union, intersection, and difference of mobjects!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "tutorial-receptor",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%manim -qm -v WARNING BooleanOperationDemo\n",
+    "\n",
+    "class BooleanOperationDemo(Scene):\n",
+    "    def construct(self):\n",
+    "        c1 = Circle(radius=2, fill_opacity=0.5, color=YELLOW).shift(LEFT)\n",
+    "        c2 = Circle(radius=2, fill_opacity=0.5, color=BLUE).shift(RIGHT)\n",
+    "        circles = VGroup(c1, c2)\n",
+    "        self.add(circles.scale(0.5))\n",
+    "        grid = VGroup(\n",
+    "            Union(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Intersection(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Difference(c1, c2, color=GREEN, fill_opacity=0.5),\n",
+    "            Exclusion(c1, c2, color=GREEN, fill_opacity=0.5)\n",
```

Comment:
Not sure if it's worth adding some complexity, but some repetition can be reduced by adding:

```py
from manim.mobject.boolean_ops import _BooleanOps

_BooleanOps.set_default(color=GREEN, fill_opacity=0.5)
```

at the top of the example. Same can also be done for `Circle`.

Reply:
Done in af0e905, but instead of changing the defaults of the somewhat arcane `_BooleanOps`, I changed it for `VMobject`.